SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Reporting a Vulnerability
```

Comment:
Content of file doesn't look correct.
Kindly follow [TRG](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01/#security-file)